ai-hub/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.11
+
+# Steps are cached with Kaniko in GCP Artifcat registry
+# The cache is kept 72 hours (see cloudbuild.yaml)
+
+WORKDIR /src
+RUN pip install poetry
+
+# Copy pyproject.toml and poetry.lock
+COPY ./pyproject.toml /src/pyproject.toml
+COPY ./poetry.lock /src/poetry.lock
+
+COPY ./phospho-python/ /phospho-python
+COPY ./backend/ /backend
+
+# Install dependencies
+RUN poetry config virtualenvs.in-project true
+RUN poetry install --no-interaction --no-ansi
+
+# Copy source code
+COPY ./ai_hub /src/ai_hub
+COPY ./main.py /src/main.py
+
+EXPOSE 8080
```

Comment:
Runs on the same port as extractor, let's change this to facilitate running the full setup locally